concent_api/core/models.py

```diff
@@ -721,27 +722,19 @@ def clean(self) -> None:
                 'payer_deposit_account': 'payer_deposit_account.ethereum_address '
                                          'cannot be the same as payee_ethereum_address'
             })
-        if not isinstance(self.payer_deposit_account, DepositAccount):
-            raise ValidationError({
-                'payer_deposit_account': 'payer_deposit_account.ethereum_address must be string'
-            })
-        if self.payee_ethereum_address == self.payer_deposit_account.ethereum_address:
-            raise ValidationError({
-                'payee_ethereum_address': 'Address of the Ethereum account belonging to the entity '
-                                          '(requestor, provider or Concent) who is supposed to receive the claim. '
-                                          'Cannot be the same as payer_deposit_account.ethereum_address'
-            })
         if not isinstance(self.payee_ethereum_address, str) or len(self.payee_ethereum_address) != ETHEREUM_ADDRESS_LENGTH:
             raise ValidationError({
                 'payee_ethereum_address': f'Address of the Ethereum account belonging to the entity '
                                           f'(requestor, provider or Concent) must be string and must be exactly '
-                                          f'{ETHEREUM_ADDRESS_LENGTH} characters.'
+                                          f'{ETHEREUM_ADDRESS_LENGTH} characters long.'
             })
         if not isinstance(self.amount, int) or self.amount <= 0:
             raise ValidationError({
                 'amount': 'Amount must be integer and be greater than 0'
             })
-        if not (self.tx_hash is None or isinstance(self.tx_hash, str)) or len(self.tx_hash) != 64:
+        if self.tx_hash is not None and \
```

Comment:
Please fix line break styling here.

Reply:
@rwrzesien I quickly skimmed through https://github.com/Code-Poets/coding-style/wiki/python-style and I am afraid I don't know what you mean :(